src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
 
+# Checkout a branch
+function checkout_branch_silently {
```

Comment:
I'd implement the "silent" aspect as a parameter to a `checkout_branch` function.

```bash
function checkout_branch {
  $silent=$1
```

Then `run_command` also gets such a parameter, and we are good to go.